Cargo.toml

```diff
@@ -45,45 +45,44 @@ required-features = ["integration-test-bin"]
 test = true
 
 [dependencies]
-anyhow = "1.0.51"
-assert_cmd = "2.0.2"
+anyhow = "1.0.56"
+assert_cmd = "2.0.4"
```

Comment:
btw I've been developing `snapbox` which is like `insta`+`assert_cmd`+`assert_fs`.  Right now, I'm using it in clap and cargo-edit and a PR is about to be merged to use it in cargo.  It is also the core of `trycmd`.

As the author of `assert_cmd` and `assert_fs`, everywhere I've used those I've now switched to either `snapbox` or `trycmd`.